Source/WebKit/UIProcess/ios/fullscreen/WKFullScreenViewController.mm

```diff
@@ -553,6 +555,7 @@ - (void)loadView
         [_cancelButton setConfiguration:cancelButtonConfiguration];
 
 #if PLATFORM(VISION)
+        // FIXME: I think PLATFORM(VISION) is always true when `alternateFullScreenControlDesignEnabled` is true.
```

Comment:
Not for people who are debugging on other platforms! But this should probably use the runtime check, not the build-time one.